extension/json/src/include/json_export.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#include "function/export/export_function.h"
+#include "function/table/bind_data.h"
+
+namespace kuzu {
+namespace json_extension {
+
+struct JsonExportFunction {
+    static constexpr const char* name = "COPY_JSON"; // common::EXPORT_JSON_FUNC_NAME;
```

Comment:
why do you add a comment like this?

Reply:
I wasn't sure if I should put the name inside common at this time, but it seems like `COPY FROM` binding searches for table functions named `COPY_{extension name}`, so no constant inside the core is necessary. I'll remove the comment.